flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/SqlCreateTableConverter.java

```diff
@@ -193,10 +179,7 @@ private CatalogTable createCatalogTable(SqlCreateTable sqlCreateTable) {
 
         return catalogManager.resolveCatalogTable(
                 CatalogTable.of(
-                        mergedSchema.toSchema(comments),
```

Comment:
After this removed, `org.apache.flink.table.api.TableSchema#toSchema(java.util.Map<java.lang.String,java.lang.String>)` seems not used and maybe can also be removed

Reply:
But it has been exposed in 1.17 as a `@PublicEvolving` API, and it's a breaking change if we remove it. I think we can leave it alone because `TableSchema` is deprecated. We can remove it entirely once `TableSchema` is dropped.